packages/lisk-p2p/src/response_handler.ts

```diff
@@ -69,10 +68,35 @@ export const instantiatePeerFromResponse = (peer: unknown): PeerConfig => {
 		ipAddress: rawPeer.ip,
 		wsPort,
 		height,
-		id,
 		os,
 		version,
 	};
 
 	return peerConfig;
 };
+
+interface RPCPeerListResponse {
+	readonly peers: ReadonlyArray<object>;
+	readonly success?: boolean; // Could be used in future
```

Comment:
isn't this current protocol schema which is used now?

Reply:
yes, as we want it to be compatible with the current protocol. Especially for `list` RPC because it has different response type than others
`{success: boolean, peers: Array<Peers>}`